provider/ocm_policies/classic/ocm_policies_state.go

```diff
@@ -36,6 +36,7 @@ type OperatorRolePolicies struct {
 type AccountRolePolicies struct {
 	Installer            types.String `tfsdk:"sts_installer_permission_policy"`
 	Support              types.String `tfsdk:"sts_support_permission_policy"`
+	SupportRhSreRole     types.String `tfsdk:"sts_support_rh_sre_role"`
```

Comment:
Surely, so what happens here is that we have only been capturing the permission policies. But it is also possible possible to fetch the trust policies through the API. This role is a dynamically generated role that needs to be added to the trust policy of the customer's support role as a principal so SRE can access when required to support the cluster.
Because TF is generating the trust policy details itself instead of fetching from the API this role is never added.